### ISSUE TITLE
Fullscreen Problem

### ISSUE THREAD

**Issue by tomads** [lhc70000/iina#1972]
**System and IINA version:**

- macOS 10.13.6
- IINA 1.00 Beta4

**Expected behavior:**
Should go automatically to fullscreenmode, when opening Video (by double-clicking video in Finder)

**Actual behavior:**
Won`t go to fullscreenmode automatically, even if the option is set in Prefs/General/Open File -> Fullscreen

**How often does this happen?**
always, not with 1.00 beta3

**Comment by lhc70000** [comment]
Please test if this build fixed the problem:

http://dl.iina.io/IINA.v1.0.0-beta4-build92.dmg

**Comment by GraOrigin** [comment]
@lhc70000 yes this does fix the fullscreen problem - for me at least. Thank's!

**Comment by tomads** [comment]
fixed.
THX
